code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -830,13 +830,13 @@
 	reagents.add_reagent(TOXIN, 2)
 	bitesize = 2
 
-/obj/item/weapon/reagent_containers/food/snacks/faggot
-	name = "faggot"
+/obj/item/weapon/reagent_containers/food/snacks/meatball
```

Comment:
The greatest loss is definitely what-must-now-be-called-meatball. God speed round ball of mystery meat.